bentoml/server/gunicorn_server.py

```diff
@@ -82,3 +83,7 @@ def load(self):
         bento_service = load(self.bento_archive_path)
         api_server = BentoAPIServer(bento_service, port=self.port)
         return api_server.app
+
+    def run(self):
+        track_server('gunicorn')
```

Comment:
just add an attribute "num_of_workers" here in the amplitude message, you can access self. num_of_workers